source/Cute/Commands/Type/TypeCloneCommand.cs

```diff
@@ -1,17 +1,151 @@
-﻿using Cute.Commands.BaseCommands;
+﻿using Contentful.Core.Models;
+using Cute.Commands.BaseCommands;
 using Cute.Commands.Login;
 using Cute.Config;
+using Cute.Constants;
 using Cute.Lib.Contentful;
+using Cute.Lib.Contentful.BulkActions.Actions;
+using Cute.Lib.Exceptions;
 using Cute.Services;
+using Newtonsoft.Json.Linq;
+using Spectre.Console;
 using Spectre.Console.Cli;
+using System.ComponentModel;
 
 namespace Cute.Commands.Type;
 
-public class TypeCloneCommand(IConsoleWriter console, ILogger logger, ContentfulConnection contentfulConnection,
-    AppSettings appSettings) : BaseLoggedInCommand<LoggedInSettings>(console, logger, contentfulConnection, appSettings)
+public class TypeCloneCommand(IConsoleWriter console, ILogger<TypeCloneCommand> logger, ContentfulConnection contentfulConnection,
+    AppSettings appSettings, HttpClient httpClient) : BaseLoggedInCommand<TypeCloneCommand.Settings>(console, logger, contentfulConnection, appSettings)
 {
-    public override Task<int> ExecuteCommandAsync(CommandContext context, LoggedInSettings settings)
+    private readonly HttpClient _httpClient = httpClient;
+    public class Settings : LoggedInSettings
     {
-        throw new NotImplementedException();
+        [CommandOption("-c|--content-type-id <ID>")]
+        [Description("Specifies the content type id to generate types for. Default is all.")]
+        public string ContentTypeId { get; set; } = null!;
+
+        [CommandOption("--source-environment-id")]
+        [Description("Specifies the source environment id.")]
+        public string SourceEnvironmentId { get; set; } = default!;
+
+        [CommandOption("-p|--publish")]
+        [Description("Whether to publish the created content or not. Useful if no circular references exist.")]
+        public bool Publish { get; set; } = false;
+
+        [CommandOption("-b|--entries-per-batch")]
+        [Description("Number of entries processed in parallel.")]
+        public int EntriesPerBatch { get; set; } = 5;
+    }
+
+    public override async Task<int> ExecuteCommandAsync(CommandContext context, Settings settings)
+    {
+        string contentTypeId = settings.ContentTypeId;
+
+        if (!ConfirmWithPromptChallenge($"destroy all '{contentTypeId}' entries in {ContentfulEnvironmentId}"))
```

Comment:
Sentence should start with an uppercase letter

Reply:
Inside of ConfirmWithPromptChallenge passed string is concatenated with hardcoded "About to" and right now it looks like this:
![image](https://github.com/user-attachments/assets/5d488a01-a0b5-4206-ae97-6f4a2d056a76)

is your comment still applicable?